src/data/nsynth.py

```diff
@@ -1,10 +1,10 @@
 import tensorflow as tf
-import preprocess
+import data.process as pro
 
 def nsynth_from_tfrecord(nsynth_tfrecord_path):
     dataset = tf.data.TFRecordDataset([nsynth_tfrecord_path])
-    return preprocess.pipeline([
-        preprocess.parse_tfrecord({
+    return pro.pipeline([
+        pro.parse_tfrecord({
             "note_str": tf.io.FixedLenFeature([], dtype=tf.string),
             "pitch": tf.io.FixedLenFeature([1], dtype=tf.int64),
```

Comment:
In my experience, shape `[]` tends to be more comfortable to use than `[1]`.

Reply:
This has nothing to do with this pull request, create a separate issue